framework/test/jest/unit/specs/modules/chain/blocks/default_account.js

```diff
@@ -15,16 +15,16 @@
 'use strict';
 
 const genesisAccount = {
```

Comment:
maybe format issue? can you revert it back if the format is correct?

Reply:
This was automatically applied by prettier/eslint so I guess format was wrong before?